WalletWasabi/WabiSabi/Backend/Rounds/RoundParameters.cs

```diff
@@ -58,5 +66,16 @@ public class RoundParameters
 		public TimeSpan OutputRegistrationTimeout { get; }
 		public TimeSpan TransactionSigningTimeout { get; }
 		public TimeSpan BlameInputRegistrationTimeout { get; }
+
+		public static RoundParameters CreateBlameRoundParameters(
+			WabiSabiConfig wabiSabiConfig,
+			Network network,
+			WasabiRandom random,
+			FeeRate feeRate,
+			Round blameOf,
+			Prison prison)
```

Comment:
@nothingmuch are you saying to replace Prison with `IEnumerable<OutPoint> allowedInputs`?